lib/prmd/templates/schemata/helper.erb

```diff
@@ -163,7 +163,7 @@
       else
         "`#{value['example'].to_json}`"
       end
-    elsif value['type'] == ['array'] && value.has_key?('items')
+    elsif (value['type'] == ['array'] && value.has_key?('items')) || value.has_key?('enum')
```

Comment:
I think maybe adding a `.to_json` here, similar to the part before, might solve the quoting issue you mentioned?